### ISSUE TITLE
Setup and Crashing

### ISSUE THREAD

**Issue by seanklei10** [Sniffdog/Sniffdogminer#90]
1) I currently have 2x 1080TI's in one computer along side another k4200 graphics card that I use as primary so the computer still works. I'd like to mine only on the two GPUs (not the k4200 card or CPU). Is there a setting for this?

2) ccminer continues to crash when its running. I have no overclocking setup and nothing else is running on the computer. 

3) when I setup my worker it asks for both username and worker. So for username i put USERNAME and is worker USERNAME.Worker like its listed on multipool or just the unique identifier? Also, after how long should that worker show up as active on multipool?

4) can I adjust the intensity of individual algos? I know when I'm typically running PHI on intensity 25 I get ~40mh/s as opposed to the 30mh/s I'm seeing when its defaulting to 20 for intensity. 

Thanks again for what looks to be a great program, can't wait to get it working!

**Comment by Sniffdog** [comment]
of course you can... all intensities are at the miners folder and Alexis needs these intensities....

I would suggest though that if your wanting to split your rig up... you should use Hashsynminer

**Comment by Sniffdog** [comment]
I also checked what you mentioned about intensity at 25? Even at 24 you start to get errors... here is a screenshot with Tpruvot 2.2.4 phi...

![screenshot 26](https://user-images.githubusercontent.com/30452757/35247918-5a5a33c8-ff89-11e7-968f-2211eb689658.png)

and this was stable at -i 23 Tpruvot 2.2.4 phi

ccminerphi1.0 at -i 23

![image](https://user-images.githubusercontent.com/30452757/35248229-99c34512-ff8a-11e7-9752-cec7f027ca63.png)

at -i 24 ccminerphi1.0

![screenshot 29](https://user-images.githubusercontent.com/30452757/35248336-fbe74cfc-ff8a-11e7-904d-fa3e8f075589.png)

at -i 25 

![screenshot 31](https://user-images.githubusercontent.com/30452757/35248447-5e2cf22c-ff8b-11e7-9752-ae57edbab664.png)

**Comment by Sniffdog** [comment]
ill leave at -i 24 on ccminerphi1.0 for a while and see if it crashes on 1080 and 1070

**Comment by seanklei10** [comment]
Thanks for all the info! I'll give Hashsynminer a try. Any thoughts on how I'm supposed to setup my workername? My username is djsmining and it says my workername is djsmining.1 so do I just put in the 1 or djsmining.1 into the .bat file for workername?

**Comment by seanklei10** [comment]
Also, why did you choose to change from Alexis78 to Tpruvot Alexis? I get ~850mh/s with tpruvot and 1050mh/s with Alexis78, for DGB sklein

**Comment by Sniffdog** [comment]
not sue what your doing wrong... it should be the same... since it's a newer version... i had no differences with mine on skein... you will have to show screenshots to prove your at those hashes... and also, show if your overclocking...

at username put djsmining

at workername put 1

password at miningpoolhub should be left at x

**Comment by seanklei10** [comment]
I'll shoot some screenshots after hashsynminer finishes benchmarks. 

A few questions though, if i only want to mine on GPU0 and GPU2 and no CPU do I just leave in 
-SplitSniffCC0 '0' -SplitSniffCC2 '2' and delete the rest?

Should I put my BTC wallet in for all 7 -Wallet's? 

 -type nvidia, should I remove any of those?

**Comment by Sniffdog** [comment]
just use only one ...... -SplitSniffCC0 '0,2' and SplitSniffEWBF '0 2'

**Comment by seanklei10** [comment]
Perfect thank you. I'm running it now. I noticed there are several powerscript files for alexis, do I have to edit each one to adjust the intensity or just 0 and 2 since that is all I'm using? 

How long should it take to finish "writing web request"?

**Comment by Sniffdog** [comment]
if your only using one then it will only download one miner.... therefore you will only need to edit one

**Comment by seanklei10** [comment]
![image](https://user-images.githubusercontent.com/35706768/35253080-96556f28-ffb1-11e7-9c07-9ffd52345e0a.png)
any reason I would error out trying to use poly?

**Comment by Sniffdog** [comment]
yeah if you look at the main window you will see nothing showing its direction in the "running" phase

it means that the api is not picking up on that algo cause it's not in miningpoolhub

**Comment by seanklei10** [comment]
So would you recommend I setup to use all the mining pools and change the code to -PoolName zpool,Hashrefinery,MPH,ahashpool ?

**Comment by Sniffdog** [comment]
no you showed me earlier what you wanted to mine in MPH.... but if you wanted to open up to all the pools then yes but make sure you have your own bitcoin address in the hashsynminer.bat

**Comment by seanklei10** [comment]
Thanks, will i need to register at each of those? Or does multipool cover it?

**Comment by seanklei10** [comment]
Below you can see alexis78 improvement over Tpruvot. This is running without any overclock on EVGA 1080ti hybrid cards

![alexis78](https://user-images.githubusercontent.com/35706768/35253522-de19ecc4-ffb3-11e7-9f2a-0b914cadb967.PNG)
Alexis78

![tpruvot](https://user-images.githubusercontent.com/35706768/35253523-dfa736fa-ffb3-11e7-852d-4865dcd8de3c.PNG)
Tpruvot

**Comment by Sniffdog** [comment]
why do you have Tpruvot open?

it's set in Alexis in 3.9 and in hashsynminer

**Comment by seanklei10** [comment]
Is it possible for it to skip? or do I have to manually take out that algo to get past it?

**Comment by Sniffdog** [comment]
you can do it on the fly.... just go into the stat folder and delete the file that has the miner and algo.. hashsynminer will re-bench

**Comment by Sniffdog** [comment]
I think that new ccminerphi1.0 has porting issues... I may have to use Tpruvot 2.2.4 for phi in Hashsynminer

**Comment by seanklei10** [comment]
Really appreciate all your help. 

Is there anything else I'll need to setup? I have my BTC wallet in, multipool worker created and its running on the correct cards. Anything else to monitor or make sure autopayouts are working? 

Do you ever run overclocking with hashsynminer?

**Comment by seanklei10** [comment]
![image](https://user-images.githubusercontent.com/35706768/35254303-a2b601dc-ffb7-11e7-94e5-37eec01c8b0f.png)
ccminer phi-1.0 seems to be working

**Comment by seanklei10** [comment]
![image](https://user-images.githubusercontent.com/35706768/35254418-1ab4f17a-ffb8-11e7-9d25-fe86c123e561.png)
also ran into this error on c11

**Comment by Sniffdog** [comment]
probably needs an intensity setting and lower than what it is running at

**Comment by seanklei10** [comment]
So far that appears to allow it to run c11, thanks!

How can I force it to rebenchmark an algo? My X17 and PHI seem way off. x17 i really only get like 40mh/s

![image](https://user-images.githubusercontent.com/35706768/35255128-7b132764-ffbb-11e7-9b6c-62bf28a910de.png)

**Comment by Sniffdog** [comment]
go into stats folder and delete ccmineralexis-x17.txt

**Comment by seanklei10** [comment]
Will that ever update as it mines or it is a onetime benchmark?

**Comment by Sniffdog** [comment]
that will update

**Comment by seanklei10** [comment]
![image](https://user-images.githubusercontent.com/35706768/35255411-e0261a02-ffbc-11e7-8482-a8ff9d2bbdee.png)
 Looks like everything is working perfect now. Thanks for your help! Is there anything else I need to setup for auto BTC payout?

**Comment by Sniffdog** [comment]
nah all is good if you didn't change the c=BTC in hashsynminer.bat and are using a btc address at zpool,hashrefinery,and ahashpool...

At miningpoolhub yes you will have to go in and do your settings for auto exchanging coins...

Glad I could help you out... hashsynminer is a bigger program and a lot harder to setup but well worth it when you are wanting to split how your computer functions

Another tip... if you get cut off from gaming having this program running in the background... just open a new user in windows and run the miner in that then come back to your original user :)

Good luck!

**Comment by seanklei10** [comment]
Is this the best place to chat with you? Do you have a discord or anything like that? 

Also, what is the point of delay? Seems just like a gap between opening the miners? Sounded like you were working on a way to get the miners to overlap possibly?

**Comment by Sniffdog** [comment]
overlap? No... porting yes... delay is the amount of time it take for a card to begin new work... this usually cools down a card a little before starting up a new algo... it's measured in seconds

**Comment by Sniffdog** [comment]
turn off ccminerphi1.0 for now and use Tpruvot 2.2.4 for phi and set intensity -i 23 please.... 

Looks like on my split rig that ccminerphi1.0 is not reporting properly

**Comment by seanklei10** [comment]
Where do you go to turn off ccminerphi1.0? What are you seeing as an issue? 

Also, what is "profit analyser"?

**Comment by Sniffdog** [comment]
previous one running provided it's on the same pool

peeps wanted to something to check each pool for profit... so i added profit analyser... works with zpool,hashrefinery,ahashpool... you can split your rig to each pool and check out an algo to see which pool is most profitable
